src/components/Reports/MPGAIncomeExpensesReport/TotalsContext/TotalsContext.tsx

```diff
@@ -35,56 +36,40 @@ export const TotalsProvider: React.FC<TotalsContextProps> = ({
   children,
   data,
 }) => {
-  const incomeTotal = useMemo(() => {
-    return sum(data.income);
-  }, [data.income]);
-
-  // const ministryTotal = useMemo(() => {
-  //   return sum(data.ministryExpenses);
-  // }, [data.ministryExpenses]);
+  const { ministry, healthcare, assessment, other } = useExpenseCategories(
+    data.expenses,
+  );
 
-  // const healthcareTotal = useMemo(() => {
-  //   return sum(data.healthcareExpenses);
-  // }, [data.healthcareExpenses]);
+  const incomeTotal = useMemo(() => sum(data.income), [data.income]);
 
-  // const miscTotal = useMemo(() => {
-  //   return sum(data.misc);
-  // }, [data.misc]);
+  const ministryTotal = sum(ministry);
```

Comment:
Would it be worth moving these sums into `useExpenseCategories`, as they will stay the same unless data changes?

Reply:
Also it looks like the total is on the mock data, I assume this won't be the case when we get the real data?